dashboard/src/components/ui/v2/Switch/Switch.tsx

```diff
@@ -1,13 +1,39 @@
+import type { FormControlLabelProps } from '@/ui/v2/FormControlLabel';
+import FormControlLabel from '@/ui/v2/FormControlLabel';
 import SwitchUnstyled, {
   switchUnstyledClasses,
 } from '@mui/base/SwitchUnstyled';
 import type { SwitchUnstyledProps } from '@mui/base/SwitchUnstyled/SwitchUnstyled.types';
-
 import { styled } from '@mui/material';
-import type { ForwardedRef } from 'react';
+import type { ForwardedRef, PropsWithoutRef } from 'react';
 import { forwardRef } from 'react';
 
-export interface SwitchProps extends SwitchUnstyledProps {}
+export interface SwitchProps extends SwitchUnstyledProps {
+  /**
+   * Label to be displayed next to the checkbox.
+   */
+  label?: FormControlLabelProps['label'];
+  /**
+   * Props to be passed to the internal components.
```

Comment:
We have two different comments for similar props:

`Props to be passed to individual component slots.` for [Radio.tsx](https://github.com/nhost/nhost/pull/1414/files#diff-cab2e6a6d80963fb89d93b52ac228ab8f2e2d9d60d3d2567547da562e922ac63) and `Props to be passed to the internal components.` here. Should we standarize it, or is there a reason for the difference here?

Reply:
Yep, we should standardize this. I'm leaning towards "individual component slots" or "each slot inside the XYZ component". The latter is being used by MUI.